concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1472,7 +1474,27 @@ jobs:
               CONCOURSE_ATC_PASSWORD=$("$VAL_FROM_YAML" jobs.concourse.properties.atc.basic_auth_password concourse-manifest/concourse-manifest.yml)
               export CONCOURSE_ATC_PASSWORD
               echo Looking for non running VMs
-              ./paas-cf/concourse/scripts/bosh-cli.sh bosh vms | tee vms.txt
+              ./paas-cf/concourse/scripts/bosh-cli.sh bosh status
+      - task: test-bosh-vms
+        config:
+          platform: linux
+          image: docker:///governmentpaas/bosh-cli
+          params:
```

Comment:
These params shouldn't be needed for this task now that it doesn't use bosh-cli.